src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -25,28 +29,51 @@ type Props = {
     translations?: Translations,
 };
 
-const ActiveState = ({
-    currentUser,
-    items,
-    onCommentDelete,
-    onTaskDelete,
-    onTaskEdit,
-    onTaskAssignmentUpdate,
-    onVersionInfo,
-    translations,
-    getAvatarUrl,
-    getUserProfileUrl,
-    getMentionWithQuery,
-    mentionSelectorContacts,
-}: Props): React.Node => (
-    <ul className="bcs-activity-feed-active-state">
-        {items.map((item: any) => {
-            const { type, id, versions, permissions } = item;
+class ActiveState extends React.Component<Props> {
+    cache: any;
+
+    list: any;
+
+    constructor() {
+        super();
+        this.cache = new CellMeasurerCache({
+            fixedWidth: true,
+        });
+    }
+
+    componentDidUpdate() {
+        this.cache.clearAll();
+        this.list.forceUpdateGrid();
+    }
 
-            switch (type) {
-                case 'comment':
-                    return (
-                        <li key={type + id} className="bcs-activity-feed-comment">
+    onEditChange = () => {
+        this.cache.clearAll();
+        this.list.forceUpdateGrid();
```

Comment:
Would we want to call [this.list.recomputeRowHeights](https://github.com/bvaughn/react-virtualized/blob/master/docs/List.md#recomputerowheights-index-number), instead?